ms2query/utils.py

```diff
@@ -105,6 +103,44 @@ def get_classifier_from_csv_file(classifier_file_name: str,
     else:
         results = pd.concat(list_of_classifiers, axis=0, ignore_index=True)
 
-    results["inchi_key"] = list_of_inchikeys
-    results.rename(columns={"inchi_key": "inchikey"}, inplace=True)
+    results["inchikey"] = list_of_inchikeys
     return results
+
+
+def columns_and_order_for_output_dataframe(return_standard_columns: bool,
```

Comment:
I find the function name too long. Maybe something like `ordered_columns_for_output` would be enough?

Reply:
Now that I looked at it in more detail: Maybe the "ordered" is not realy true here and it is rather a function `column_names_for_output` or something like this, right?

Reply:
Yes I agree. I will turn it into a shorter name. The ordered was used in the name here, since this output is also used for sorting the columns in the dataframe that is created. However, I think column_names_for_output is fine.